dotnet-cookbook-redirect/README.adoc

```diff
@@ -0,0 +1,15 @@
+= DotNet Cookbook Redirect
+
+A simple NGINX web server configured to redirect:
+    from:    $scheme://dotnetcookbook.com/PATH
+    to:      $scheme://tanzu.vmware.com/developer/cookbooks/dotnet/PATH
```

Comment:
This seems nice, but there isn't a 1:1 as some content has been removed... for example /intro is gone. We've been asked to just drop everything at /cookbooks/dotnet and not worry about the rest of the path

Reply:
Luckily I misunderstood how the redirect was working and documented with that misunderstanding.

Turns out that, if running locally per docs, that:
```
http http://localhost:8080/nosuchpage
```

Goes to where we want.  That root.